modules/gapi/src/executor/gstreamingexecutor.cpp

```diff
@@ -800,8 +800,9 @@ void cv::gimpl::GStreamingExecutor::setSource(GRunArgs &&ins)
             }
         }
     };
+    bool containsOriginalMeta = gm.metadata().contains<OriginalInputMeta>();
```

Comment:
Not sure if this flag is required, see below.

Reply:
Fixed, thanks a lot!